src/pages/seoyeongYoon/Login/LoginSeo.scss

```
@@ -0,0 +1,68 @@
+.login {
+  padding: 0 15px 0 14px;
+  box-sizing: border-box;
+  .container {
+    height: 100vh;
+    display: flex;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+    .container_form {
+      height: 500px;
+      width: 480px;
+      padding: 0 36px 0 36px;
+      box-sizing: border-box;
+      border: 1px solid rgb(214, 214, 214);
```

```suggestion
.login {
  padding: 0 15px 0 14px;
  box-sizing: border-box;
  
  .container {
    height: 100vh;
    display: flex;
    text-align: center;
    justify-content: center;
    align-items: center;
    
    .container_form {
      height: 500px;
      width: 480px;
      padding: 0 36px 0 36px;
      box-sizing: border-box;
      border: 1px solid rgb(214, 214, 214);
```
scss 속성 간 가독성을 위해 한칸씩 띄워주세요!